pkg/loki/config_wrapper.go

```diff
@@ -8,7 +8,6 @@ import (
 	"time"
 
 	cortexcache "github.com/cortexproject/cortex/pkg/chunk/cache"
```

Comment:
can this be loki ?

Reply:
Just checked - we can't until we merge the `queryrange` fork.